projects/lights_control/inc/lights_simple_peripherals.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+typedef enum {
+  LIGHTS_SIMPLE_STATE_ON = 0,
+  LIGHTS_SIMPLE_STATE_OFF,
+  NUM_LIGHTS_SIMPLE_STATES,
+} LightsSimpleState;
+
+typedef StatusCode (*SimplePeripheralCallback)(Event e);
```

Comment:
Prefix with `Lights`. As mentioned in your other PR please pass `Event` by const pointer if possible. (Here and elsewhere).

Reply:
gotcha